types/src/types.ts

```diff
@@ -617,6 +619,24 @@ export interface TaskPushNotificationConfig {
 }
 // --8<-- [end:TaskPushNotificationConfig]
 
+// --8<-- [start:AgentCardSignature]
+/**
+ * AgentCardSignature represents a JWS signature of an AgentCard.
+ * This follows the JSON format of an RFC 7515 JSON Web Signature (JWS).
+ */
+export interface AgentCardSignature {
+  /**
+   * The protected JWS header for the signature. This is a Base64url-encoded
+   * JSON object, as per RFC 7515.
+   */
+  protected: string;
+  /** The computed signature, Base64url-encoded. */
+  signature: string;
+  /** The unprotected JWS header values. */
+  header?: { [key: string]: any };
+}
+// --8<-- [end:AgentCardSignature]
+
```

Comment:
This definition should probably be above the definition of `AgentCard` where it's used.